### PR TITLE
[YUNIKORN-654] Include node sorting policy in cluster page

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
Showing "node sort policy" in UI dashboard. 

### What type of PR is it?
* [ ] - Bug Fix
* [x] - Improvement
* [ ] - Feature
* [ ] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-654

### How should this be tested?
https://app.travis-ci.com/github/apache/incubator-yunikorn-web/builds/234336686

### Screenshots (if appropriate)
截圖 2021-08-02 下午3 18 20

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Review by akhilpb001 — APPROVED** [review]

**Comment by yangwwei** [comment]
thank you @kobe860219  for working on this, thank you @akhilpb001  for the review. 
the pre-commit check fails, could u pls take a look?

**Comment by kobe860219** [comment]
Hi @akhilpb001 ! There is aa error in pre-commit check. I found the problem below when `make test` :
image

I have no idea what cause it. If you available, could you give me some hints ?

**Comment by yangwwei** [comment]
seems like the travis job is still failing: https://app.travis-ci.com/github/apache/incubator-yunikorn-web/jobs/529229491.
it says: The command "yarn test:coverage" exited with 1. @kobe860219  could u pls check this? you can run the same command locally as well.

**Comment by kobe860219** [comment]
@yangwwei  It's the same error when I run the same command locally. The problem outputs the info `TypeError: this.scheduler.fetchPartitionList is not a function` . Currently, I have no idea why it occur.

**Comment by akhilpb001** [comment]
Hi @kobe860219, this is unit test failure. Add following lines to `MockSchedulerService` in `mocks.ts` file to fix this.
```
fetchPartionList: () => of([]),
```

CC: @yangwwei

**Review by akhilpb001 — CHANGES_REQUESTED** [review]
See my comment and fix broken unit test

File: src/app/app.component.ts [hunk 1]
```diff
@@ -55,11 +55,11 @@ export class AppComponent implements OnInit {
     do {
       const childrenRoutes = currentRoute.children;
       currentRoute = null;
-      childrenRoutes.forEach(route => {
+      childrenRoutes.forEach((route) => {
```
Comment (akhilpb001) [inline src/app/app.component.ts:58]: Please configure your editor (VS Code) to not to add bracket around single param in arrow function. If you are using VSCode and installed prettier, you  can configure this in prettier extension settings.

**Comment by kobe860219** [comment]
@yangwwei  The pre-commit check passed now. Thanks @akhilpb001 help.  I disabled my IDE's prettier and use `yarn prettify` to format insteed.

**Comment by yangwwei** [comment]
thank you @kobe860219 for getting this done
and thank you @akhilpb001 for the review and suggestions!

**Pull request merged**